Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM jenkins:2.19.1-alpine
 
 COPY plugins.txt /usr/share/jenkins/plugins.txt
-RUN /usr/local/bin/plugins.sh /usr/share/jenkins/plugins.txt
+RUN /usr/local/bin/install-plugins.sh $(cat /usr/share/jenkins/plugins.txt | tr '\n' ' ')
```

Comment:
I wonder if we should just rename the https://github.com/fabric8io/jenkins-docker/blob/master/plugins.txt -> https://github.com/fabric8io/jenkins-docker/blob/master/plugins.sh ?  Might make it a bit clearer what's going on.